File: codeforces/mashups/9/B.cpp
#include <bits/stdc++.h>

using namespace std;

// B. Números menores ou iguais

int main() {
    ios_base::sync_with_stdio(false); cin.tie(0); cout.tie(0);

    int n, m;

    cin >> n >> m;

    vector<int> a(n), b(m);

    for (int i = 0; i < n; i++) {
        cin >> a[i];
    }

    for (int i = 0; i < m; i++) {
        cin >> b[i];
    }

    sort(a.begin(), a.end());

    for (int i = 0; i < m; i++) {
        auto it = upper_bound(a.begin(), a.end(), b[i]);
        cout << (it - a.begin()) << " ";
    }

    cout << "\n";

    return 0;
}

File: codeforces/mashups/9/B.go
package main

import (
	"fmt"
	"sort"
)

// B. Números menores ou iguais

func main() {

	var n, m int
	fmt.Scanf("%d %d", &n, &m)

	a := make([]int, n)

	for i := 0; i < n; i++ {
		fmt.Scanf("%d", &a[i])
	}

	sort.Ints(a)

	for i := 0; i < m; i++ {
		var bi int
		fmt.Scanf("%d", &bi)

		l := 0
		r := n - 1

		ans := -1

		for l <= r {
			mid := (l + r) / 2

			if bi >= a[mid] {
				l = mid + 1
				ans = mid
			} else {
				r = mid - 1
			}
		}

		fmt.Printf("%d ", ans+1)
	}
}
